WordPress/Classes/ViewRelated/Me/MeHeaderView.h

```diff
@@ -1,20 +1,20 @@
 #import <UIKit/UIKit.h>
 
 typedef void (^MeHeaderViewCallback)(void);
-typedef void (^MeHeaderViewDropCallback)(UIImage *image);
+typedef void (^MeHeaderViewDropCallback)(UIImage * _Nonnull image);
 
 @interface MeHeaderView : UIView
 
-@property (nonatomic,   copy) NSString                  *displayName;
-@property (nonatomic,   copy) NSString                  *username;
-@property (nonatomic,   copy) NSString                  *gravatarEmail;
-@property (nonatomic,   copy) MeHeaderViewCallback      onGravatarPress;
-@property (nonatomic,   copy) MeHeaderViewDropCallback  onDroppedImage;
-@property (nonatomic, assign) BOOL                      showsActivityIndicator;
+@property (nonatomic, nonnull,  copy) NSString                  *displayName;
+@property (nonatomic, nonnull,  copy) NSString                  *username;
+@property (nonatomic, nullable, copy) NSString                  *gravatarEmail;
+@property (nonatomic, nullable, copy) MeHeaderViewCallback      onGravatarPress;
+@property (nonatomic, nullable, copy) MeHeaderViewDropCallback  onDroppedImage;
+@property (nonatomic, assign)         BOOL                      showsActivityIndicator;
```

Comment:
Nitpicky of me but, it might be nice to go ahead and just clean up all the extra whitespace between the property type and property name.  The formatting you see here is something we've been trying to move away from but there are still a few places (like this one) we hadn't gotten to yet.

Reply:
Fixed :)